Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -637,12 +641,34 @@
                         <ToolTip Content="{Binding FolderTooltipText}" />
                     </ToolTipService.ToolTip>
                 </controls:DataGridTextColumn>
+                <controls:DataGridTemplateColumn
+                    x:Name="syncStatusColumn"
+                    x:Uid="syncStatusColumn"
+                    CanUserResize="False"
+                    CanUserSort="False"
+                    DisplayIndex="2"
+                    IsReadOnly="True"
+                    Header="Status"
+                    Tag="Status"
+                    Visibility="{x:Bind local:App.CurrentInstance.InstanceViewModel.IsPageTypeCloudDrive, Converter={StaticResource BoolToVisibilityConverter}, Mode=OneWay}">
```

Comment:
Instead of referencing the view model from the xaml, would it make more sense to create the reference in the code behind of GenericFile Browser?

Reply:
Do you mean putting in BaseLayout something like:
`public InstanceViewModel Instance => App.CurrentInstance.InstanceViewModel;`
and than doing:
`Visibility="{x:Bind Instance.IsPageTypeCloudDrive`}` in GenericFileBrowser?

Reply:
@gave92 I don't think it really matters that much in this case.

Reply:
@yaichenbaum As long as they're consistent in the future